client/src/reducers/pagination.test.js

```diff
@@ -0,0 +1,68 @@
+import reducer from './pagination'
```

Comment:
Should this file be under `client/tests`?

Reply:
For now I've added the according reducer and action tests in the client/tests folder.
However, when keeping the tests close to the file it is easier to quickly see which files do NOT have any tests yet.
Could be interesting to take this in to account in the future